core/src/main/java/org/apache/iceberg/RemoveSnapshots.java

```diff
@@ -60,12 +62,16 @@ public void accept(String file) {
     }
   };
 
+  // Creates an executor service that runs each task in the thread that invokes execute/submit.
+  private static final ExecutorService defaultDeleteExecutorService = MoreExecutors.newDirectExecutorService();
```

Comment:
Nit: Static final names should be `ALL_CAPS`.

Reply:
And can you move this near the logger instead of here? That will keep all the static final fields together.

Reply:
Right, didn't run the checks, I'll run the gradle checks before submitting the changes.